actors/miner/src/expiration_queue.rs

```diff
@@ -78,17 +77,19 @@ impl ExpirationSet {
         on_time_pledge: &TokenAmount,
         active_power: &PowerPair,
         faulty_power: &PowerPair,
-    ) -> anyhow::Result<()> {
+    ) -> Result<(), ActorError> {
         // Check for sector intersection. This could be cheaper with a combined intersection/difference method used below.
         if !self.on_time_sectors.contains_all(on_time_sectors) {
-            return Err(anyhow!(
+            return Err(actor_error!(
+                illegal_argument,
```

Comment:
Another interesting example case. This is illegal state in Go, and that's more correct because the cause of this is some inconsistency or broken invariants in the actor's state. This doesn't result from a bad parameter provided by top-level caller.